modules/ROOT/pages/install-aws.adoc

```diff
@@ -199,9 +199,7 @@ This step installs Runtime Fabric across all servers to form a cluster. It may t
 
 ==== Run using Docker
 
-. Navigate to the `../rtf-install-scripts/aws/` directory. You must run terraform from this directory.
-. Verify your current directory is the unzipped directory. When running the `dir` command, you should 
-see the `aws` directory listed, along with other directories (`azure`, `manual`, etc).
+. Navigate to the `../rtf-install-scripts/` directory. You must run terraform from this directory. When running the `dir` or `ls` command, you should see the `aws` directory listed along with other directories (`azure`, `manual`, etc).
```

Comment:
GLOBALLY try to make each step just one user action, or one flow of related actions, and in one sentence. If there is extra information that the user needs TO PERFORM THIS ONE STEP, then include it; otherwise move it to a more appropriate place or omit it:
```suggestion
. Navigate to the `../rtf-install-scripts/` directory.
```
The second sentence of the original is implied by the flow from nav to the next step of initializing.
The rest of the info belongs with steps where the user runes dir or ls, I think?